### DISCUSSION TITLE
How can I remove the border radius from a ToolStrip control?

### DISCUSSION THREAD

**Discussion by EliotVU** [Krypton-Suite/Standard-Toolkit#1808] [Q&A]
See this screenshot, there appears to be a rounded border radius for the KryptonToolStrip control, one I cannot seem to get rid off, I'm not sure if it's a bug or if I'm missing something:
![stripborderradius](https://github.com/user-attachments/assets/f2543ff6-6d4a-42e6-86e0-8a47c755786d)

Also (probably a bug) Changing the border radius (0 or -1) of the Dock tabs does not get rid off it.

**Comment by Smurf-IV** [comment]
FYI: The tab's "Shape" is controlled by Navigator Tab settings
